### PR TITLE
Get edge

### PR DESCRIPTION
**Pull request by ygao0719**
- [x] Top-level README “Table of Contents” is updated
 - [x] Feature tasks for this challenge are completed
 - [x] Unit tests written and passing
     - [x] “Happy Path” - Expected outcome
     - [x] Expected failure
     - [x] Edge Case (if applicable/obvious)
 - [x] README for this challenge is complete
     - [x] Summary, Description, Approach & Efficiency, Solution
     - [x] Link to code
     - [x] Picture of whiteboard

### CODE REVIEW

File: 401-code-challenges/Data-Structures/src/main/java/graph/Graph.java [hunk 1]
```diff
@@ -76,4 +76,30 @@ public int size(){
...
+        int total = 0;
+        for (int i = 0;i < arr.length-1; i++) {
+
+            int j = i + 1;
+            Edge newEdge = new Edge(arr[i],arr[j]);
+
+            if (edges.contains(newEdge)){
+                total = total + newEdge.weight;
+            }else{
+                break;
```
Comment (evanslaton) [inline 401-code-challenges/Data-Structures/src/main/java/graph/Graph.java:93]: I think you need to return `false` here

File: 401-code-challenges/Data-Structures/src/main/java/graph/Graph.java [hunk 2]
```diff
@@ -76,4 +76,30 @@ public int size(){
...
+            }
+        }
+
+        if (total == 0){
+            return "false " + 0;
+        }else{
+            return "true " + total;
```
Comment (evanslaton) [inline 401-code-challenges/Data-Structures/src/main/java/graph/Graph.java:100]: Won't this return `true` even if the whole 'flight path' isn't possible? As long as some of the cities are adjacent then `total` won't be equal to 0.
